examples/grpc_example/sources/common/ClientRequest+Convenience.swift

```diff
@@ -0,0 +1,12 @@
+// This file is copied from ClientRequest+Convenience.swift in GRPCCore
+// TODO: Request that they make this public.
```

Comment:
Please do not leave TODO comments. If it is something that needs to be followed up, please create an issue and refer to it (e.g., GHxxx where xxx is the `rules_swift_package_manager`  issue number). If we are requesting something in another repo, which is the case here, please create an issue there and post a link in a comment here.

Reply:
Sure I can, though this is just an example and not part of the core repo?
For whatever reason, it's currently difficult to make unary requests out-of-the-box in grpc-swift 2.0
Everything actually goes through bidirectional streaming and unary requests are just bidirectional streaming requests where the client makes one request and hangs up after the response. I can file an issue on their repo and link it here though.

Reply:
Updated the comment to:
// Requested they make this public: https://github.com/grpc/grpc-swift/issues/2213
// When they do, we can remove this.